src/Application/Adapters/Laravel/InsightsServiceProvider.php

```diff
@@ -22,9 +22,11 @@ public function register(): void
 
     public function boot(): void
     {
-        $this->publishes([
-            __DIR__ . '/../../../../stubs/laravel.php' => $this->app->configPath('insights.php'),
-        ], 'config');
+        if (strpos($this->app->version(), 'Lumen') !== 0) {
```

Comment:
Did you double check that this code is working as expected in a normal lararel application?

Reply:
There is a method called `version` in Laravel, and that one just returns a version number in Laravel.

However I think a "better" way to do this is to check `$this->app` implements `Illuminate\Contracts\Foundation\Application`, because then we know it's a Laravel project.

Reply:
Ah I did write the instanceof version first and changed it, I'll switch back to... always trust your initial thoughts!